packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -761,6 +761,38 @@ const TESTS = new Map([
       { test: 'notInclude', expected: 'utm_source=email' },
     ]],
   ])],
+  ['verifyCodeEmail', new Map([
+    ['subject', { test: 'equal', expected: `Verification code: ${MESSAGE.code}` }],
+    ['headers', new Map([
+      ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('verifyEmail') }],
+      ['X-Template-Name', { test: 'equal', expected: 'verifyCodeEmail' }],
+      ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.verifyCodeEmail }],
+      ['X-Verify-Short-Code', { test: 'equal', expected: MESSAGE.code }],
+    ])],
+    ['html', [
+      { test: 'include', expected: configHref('privacyUrl', 'welcome', 'privacy') },
+      { test: 'include', expected: configHref('supportUrl', 'welcome', 'support') },
+      { test: 'include', expected: 'Is this you signing up?' },
```

Comment:
Fwiw, I saw most value in these kinds of assertion for ensuring data has been substituted in the right position (and with the right value obviously). Also if content is inside conditional blocks. So this assertion I'm not sure is adding a great deal...